client/extensions/woocommerce/state/selectors/promotions.js

```diff
@@ -44,6 +44,11 @@ export function getPromotionsPerPage( rootState ) {
 	return list.perPage;
 }
 
+export function getPromotionsSearch( rootState ) {
+	const list = get( rootState, [ 'extensions', 'woocommerce', 'ui', 'promotions', 'list' ], {} );
+	return list.searchFilter;
```

Comment:
The old searchFilter in the components defaulted to '', and it also does in the `list-reducer` should this selector return an empty string here too instead of null if no filter is defined?

Reply:
Also should we have some test coverage for this new selector?

Reply:
I added the default in the selector and tests in ea1c696